src/tools/fuzzing.h

```diff
@@ -864,13 +885,16 @@ class TranslateToFuzzReader {
                 WeightedOption{&Self::makeGlobalGet, Important},
                 WeightedOption{&Self::makeConst, Important});
     if (canMakeControlFlow) {
-      options.add(FeatureSet::MVP,
-                  WeightedOption{&Self::makeBlock, Important},
-                  WeightedOption{&Self::makeIf, Important},
-                  WeightedOption{&Self::makeLoop, Important},
-                  WeightedOption{&Self::makeBreak, Important},
-                  &Self::makeCall,
-                  &Self::makeCallIndirect);
+      options
+        .add(FeatureSet::MVP,
+             WeightedOption{&Self::makeBlock, Important},
+             WeightedOption{&Self::makeIf, Important},
+             WeightedOption{&Self::makeLoop, Important},
+             WeightedOption{&Self::makeBreak, Important},
+             &Self::makeCall,
+             &Self::makeCallIndirect)
+        .add(FeatureSet::ReferenceTypes | FeatureSet::ExceptionHandling,
```

Comment:
why is reference types here? (i would expect just exceptions)

Reply:
EH feature needs reference type feature to be enabled because of the existence of `exnref` type. We don't currently have a way of specifying dependencies between features in Binaryen, so I'm specifying both features everywhere I need the EH feature. If people are using clang (or Emscripten, which uses clang) to generate wasm binaries that are fed to Binaryen, clang will take care of setting both features when they use `-fwasm-exceptions`, so there won't be clang-generated binaries that have only EH feature set and not reference types though.

Reply:
I see, thanks.

How about maybe doing an assertion in a more central place that if EH is enabled so is reftypes? For example in the main fuzzer entry point.

Reply:
Done in the beginning of `TranslateToFuzzReader::build`.

Reply:
Turns out this errors out when fuzz_opt.py sets `--disable-reference-types` but not `--disable-exception-handling`. To prevent this we need to make fuzz_opt.py know about the dependencies, which I'm not sure a good thing to do. Currently the fuzzer is set to generate EH instructions only when both options are enabled, so I think that should be fine..? Deleted the assertion in `TranslateToFuzzReader::build` for now; please let me know if you think otherwise.

Reply:
sgtm